conf/app.ini

```diff
@@ -203,7 +203,8 @@ IMPORT_LOCAL_PATHS = false
 ; Whether to allow signin in via OpenID
 ENABLE_OPENID_SIGNIN = true
 ; Whether to allow registering via OpenID
-ENABLE_OPENID_SIGNUP = true
+; Do not include to rely on DISABLE_REGISTRATION setting
+;ENABLE_OPENID_SIGNUP = true
```

Comment:
Uhm, I didn't know about %(DISABLE_REGISTRATION) - would that use the value of DISABLE_REGISTRATION found under `custom` dir too ? 

Renaming a variable would be a compatibility break, then we'd need to migrate config files ?